packages/account-v2/src/containers/PaymentMethods/PaymentMethodField.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+import { Input } from '@deriv-com/ui';
+import { TPaymentMethod, TPaymentMethodIdentifier, TPaymentMethodInfo, TProofOfOwnershipFormValue } from 'src/types';
+import { formatIdentifier } from 'src/utils';
+import { FileUploaderField } from '../POOForm/FileUploadField';
+import { PaymentMethodInstructions } from './PaymentMethodInstructions';
+
+type TPaymentMethodFieldProps = {
+    paymentMethodDetail: TPaymentMethodInfo;
+};
+
+export const PaymentMethodField = ({ paymentMethodDetail }: TPaymentMethodFieldProps) => {
+    const formik = useFormikContext<TProofOfOwnershipFormValue>();
+
+    if (!formik) {
+        throw new Error('PaymentMethodField must be used within a Formik component');
+    }
+
+    const { errors, setFieldValue, values } = formik;
+
+    const paymentMethod = paymentMethodDetail.paymentMethod.toLowerCase() as TPaymentMethod;
+
+    const handleBlur = (
+        identifierType: TPaymentMethodIdentifier,
+        paymentID: number,
+        paymentMethodIdentifier?: string
+    ) => {
+        handleIdentifierChange(formatIdentifier(identifierType, paymentMethodIdentifier), paymentID);
+    };
+
+    const handleIdentifierChange = (paymentMethodIdentifier: string, paymentID: number) => {
+        const selectedPaymentMethod = values[paymentMethod] || {};
+
+        selectedPaymentMethod[paymentID] = {
+            ...selectedPaymentMethod[paymentID],
+            paymentMethodIdentifier,
+        };
+
+        setFieldValue(paymentMethod, selectedPaymentMethod);
+    };
+
+    return (
+        <div className='flex flex-col gap-24'>
+            <PaymentMethodInstructions paymentMethod={paymentMethod as TPaymentMethod} />
+            <fieldset className='flex flex-col gap-24'>
+                {paymentMethodDetail.items &&
+                    paymentMethodDetail.items.map(item => {
```

Comment:
can we move this  to a separate compnent

Reply:
Let me check on this